coalib/output/Logging.py

```diff
@@ -4,25 +4,50 @@
 import logging
 import logging.config
 
+from coalib.output.printers.LOG_LEVEL import LOG_LEVEL
 
-def configure_logging():
+
+def configure_logging(new_log_level=logging.INFO, incremental=False):
     """
     Configures the logging with hard coded dictionary.
+    When changing the log_level, the incremental param should be True.
+    json-handler are created so that they exist when
+    the Log Level is changed via `configure_logging`
+
+    :param new_log_level: The new log level to set for json and console
+                          handlers.
+    :param incremental: `False` if setting up new loggers, `True` if only
```

Comment:
We use RST, so it's double back-ticks :)

Also, the two paramdocs are weirdly aligned.